src/components/PageWithSideMenu.tsx

```diff
@@ -55,19 +56,20 @@ export const SideNavHeader = ({ icon, text, id }: SideNavHeaderProps) => {
 };
 
 export const NavigationList = ({ sx, ...props }: BoxProps) => (
-  <nav>
-    <Box
-      sx={{
-        pt: '0.5rem',
-        display: 'flex',
-        flexDirection: 'column',
-        gap: '0.5rem',
-        a: { textTransform: 'none' },
-        ...sx,
-      }}
-      {...props}
-    />
-  </nav>
+  <Box
+    component="nav"
```

Comment:
Usikker på hvor vi ønsker å plassere `<nav>` i vårt tilfelle altså 🤔 
Kommer ikke på noen andre sider med lik struktur vi kan sjekke opp mot heller. I denne runden er dette uansett en forbedring da, så ingen blokker her, men ser for meg vi ønsker å flytte den ett hakk ut slik at alle accordions blir under `<nav>` en gang i nær fremtid